spec/features/visitor_sees_show_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+feature "visitor visits the show page" do
+  scenario "details are on the page" do
+
```

Comment:
this line is unnecessary. Since the line after the `do` is the first line of the actual test.